github-runner-manager/tests/unit/platform/test_github_provider.py

```diff
@@ -0,0 +1,117 @@
+#  Copyright 2025 Canonical Ltd.
+#  See LICENSE file for licensing details.
+
+
+"""Test for the github provider module."""
+
+from unittest.mock import MagicMock
+
+import pytest
+
+from github_runner_manager.github_client import GithubClient
+from github_runner_manager.manager.models import InstanceID, RunnerMetadata
+from github_runner_manager.platform.github_provider import (
+    GithubRunnerNotFoundError,
+    GitHubRunnerPlatform,
+)
+from github_runner_manager.types_.github import GitHubRunnerStatus, SelfHostedRunner
+
+
+def _params_test_get_runner_health():
+    """Parameterized data for test_get_runner_health."""
+    prefix = "unit-0"
+    runner_id = 3
+    metadata = RunnerMetadata(platform_name="github", runner_id=str(runner_id))
+    instance_id = InstanceID.build(prefix=prefix)
```

Comment:
nit: can you add "instance_id,metadata,self_hosted_runner,expected_online,expected_busy,expected_deletable" as a comment here to quickly understand the params without having to scroll down? Thank you.